docs/reference/quick-start.asciidoc

```diff
@@ -0,0 +1,284 @@
+[[quick-start]]
+== Quick Start
+
+One of the best ways to learn about Elasticsearch is to use it. This page walks
+you through steps for running {es} and doing common tasks. By the end, you
+should know how to:
+
+* Add your data to {es}
+* Run basic searches
+* Update or delete data in {es}
+
+Let's get started!
+
+[float]
+=== Install and run {es}
+
```

Comment:
This section reminds me a lot of https://www.elastic.co/guide/en/elastic-stack-get-started/master/get-started-elastic-stack.html#install-elasticsearch

If you agree, perhaps we can combine the content and just re-use it in both places (e.g. via a tagged region).  I'm happy to help with that.

Reply:
That sounds great! I'll go ahead and add a tag to this section. I'll look into combining the two docs as part of another issue/PR.